Market Maker Guide.md

```diff
@@ -0,0 +1,256 @@
+This guide is written for individuals and professionals looking to become a market maker on OpenDEX.
+
+# Supported Networks
+
+## Simnet
+Private chains which are maintained by Exchange Union. We’ll automatically open channels to you and push over some coins, you’ll be trading against our bots and anyone else running simnet. It’s the perfect monopoly-money-playground to see how things work and play around with `xucli` commands. It’s easy: run one script, wait for about 15 minutes and you are ready to go. **You want to start with this!**
+
+Status: `live` | Setup time: `~15 mins` | Required disk space: `<1 GB`
+
+## Testnet
+bitcoin testnet 3, litecoin testnet 4, ethereum ropsten. Faucets: [t-BTC](https://coinfaucet.eu/en/btc-testnet/), [t-LTC](https://faucet.xblau.com/), [t-ETH 1](https://faucet.ropsten.be/) & [2](https://faucet.metamask.io/). Quite a bit of manual work to be done here. If you need help or some channels with testnet coins, hit us up on [Discord](https://discord.gg/YgDhMSn)!
+
+Status: `live` | Setup time: `5-24h` | Required disk space (when using full nodes): `120 GB`
+
+## Mainnet
+Real money. Only with #reckless hat.
+
+Status: `live` | Setup time: `1-3 days` | Required disk space (when using full nodes): `700 GB`
+
+
+# Requirements
+
+1. Linux or macOS. [Windows WSL 2](https://docs.microsoft.com/en-us/windows/wsl/wsl2-install) support is currently experimental and not tested regularly. This guides assumes ubuntu 18.04.
+
+2. Hardware with minimum 12GB RAM and a SSD for testnet/mainnet if you want to avoid using [infura](https://infura.io/). The more, the better - `geth` likes RAM. A lot. If you have >=24GB RAM available, you can significantly shorten `geth`'s syncing time by increasing its `--cache=1024` flag for [testnet](https://github.com/ExchangeUnion/xud-docker/blob/master/xud-testnet/docker-compose.yml) or [mainnet](https://github.com/ExchangeUnion/xud-docker/blob/master/xud-mainnet/docker-compose.yml) to something larger. Since market makers should be online 24/7 and we are ushering in a post-cloud era, we recommend setting up a power-efficient linux box connected to a stable internet connection. The hardware requirements for this box differ: if you are ok connecting to [infura](https://infura.io/), or some other external geth node, a Pi4 is enough. If you want to run all full nodes yourself, a 16GB RAM+2TB HHD+32 GB SSD thin client does the job. A small SSD is enough, see [Tips 'n tricks](https://docs.exchangeunion.com/start-trading/user-guide#tips-n-tricks) below on how to split data between an HDD and SSD. Read about root cause [here](https://medium.com/blockchain-studio/ethereum-client-geth-v1-9-0-released-whats-new-2b3de043ee16). Hardware guide **coming soon**!
```

Comment:
Is "thin client" here sarcastic or a typo? :stuck_out_tongue: For splitting bitcoin data between HDD and SDD, you can also recommend the `blocksdir` config option.

Reply:
> Is "thin client" here sarcastic or a typo?
No, just me being .. tired. Fixed to "device".

Reply:
> For splitting bitcoin data between HDD and SDD, you can also recommend the blocksdir config option.

I don't think there is the need to split anything for bitcoin, it can just all run on HDD. Added this. Also, we don't make it easy to use `blocksdir` atm since it's all in a container, so I left it out.

Reply:
Sounds good, thanks.